tower-balance/src/p2c/service.rs

```diff
@@ -38,14 +42,20 @@ pub struct Balance<D: Discover, Req> {
     rng: SmallRng,
 }
 
+impl<D: Discover, Req> Unpin for Balance<D, Req> {}
```

Comment:
Probably not okay?

Reply:
@jonhoo discover doesn't take pin so it must box so this is safe since there is no unsafe code.

Reply:
Hmm, but then why do you need this `impl` at all?